### ISSUE TITLE
Click event on Timeline

### ISSUE THREAD

**Issue by fionasanoif** [almende/vis#556]
Hi,

I am trying to figure out the group that is clicked on the timeline. Not on the Item but the background.

Is there any existing event that handles this. Click the timeline and return the group that was clicked?
I would really appreciate any help.

Fiona

**Comment by josdejong** [comment]
This is currently not supported, but it's on the to-do list to add for example click handlers to the Timeline. 

It would indeed be useful to provide both item and group in these event handlers.

**Comment by AlexDM0** [comment]
Would: 
```
window.onclick = function (event) {
    console.log(event);
}
```

not suffice?

**Comment by josdejong** [comment]
The just released version 3.11.0 contains new events `click`, `doubleClick`, and `contextmenu`. The callbacks are invoked with a properties object containning useful information about where the click happened. 

See docs: 
http://visjs.org/docs/timeline.html#Events
http://visjs.org/docs/timeline.html#getEventProperties
